ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--
+
+Hello! 
+Please take a quick look at those things down there. They're quite important.
+Really! We wrote them for you. Yes, you! With utmost care. Read them.
+If you're filing a bug, please include every step so as to help us reproduce it on our machines. If you're unsure about how to file an issue, use the issue template. If you need any help regarding usage of coala, check out the documentation or hit us up on chat. You can ignore or delete this text, it is commented and won't appear when the issue is submitted or previewed.
+
+Chat: https://coala.io/chat
+Documentation: https://docs.coala.io
+
+-->
+
+Filing Issues
```

Comment:
would be better if you copy the template used in the repo `coala/coala` https://github.com/coala/coala/blob/master/ISSUE_TEMPLATE and provide a link to the content following this line i.e. at https://github.com/coala/coala/blob/master/CONTRIBUTING.rst#filing-issues instead of dumping it over here

Reply:
the content inside the `<!-- -->` is fine

Reply:
would be better to add a statement regarding adding links of referenced files and gitter conversations, GitHub comments in case the issue is better explained that way

Reply:
@ishanSrt, should the above statement be added in `<!-- -->` or in the Markdown style?

Reply:
look at the same file in the coala repo, everything's within `<!-- -->`